bin/vyper

```diff
@@ -78,6 +64,13 @@ def get_source_map_asm_list(code):
 
 
 if __name__ == '__main__':
+    
+    warnings.simplefilter('always')
+    sys.tracebacklimit = 0
+    tb_limit = os.environ.get('VYPER_TRACEBACK_LIMIT')
+    if tb_limit:
+        sys.tracebacklimit = int(tb_limit)
```

Comment:
Considering these are used in a global scope, I think having them outside main might make sense?

Reply:
Correct, these are all related to CLI usage ;)